packages/core/src/components/hotkeys/hotkey.tsx

```diff
@@ -83,7 +84,7 @@ export class Hotkey extends AbstractPureComponent<IHotkeyProps, {}> {
     };
 
     public static isInstance(element: any): element is React.ReactElement<IHotkeyProps> {
```

Comment:
borrow this type guard signature for your util function please.

and also remove this function since you replaced its usage (in Hotkeys below) with the new util function.